resources/js/processes/screen-builder/typeForm.js

```diff
@@ -4,12 +4,11 @@ import FileDownload from "./components/file-download";
 import FileUpload from "./components/form/file-upload";
 import initialControls from "@processmaker/spark-screen-builder/src/form-builder-controls";
 import VueFormElements from "@processmaker/vue-form-elements";
+import {renderer} from "@processmaker/spark-screen-builder";
 
 Vue.use(VueFormElements);
 
-import { renderer } from "@processmaker/spark-screen-builder";
-
-let FormText = renderer.FormText;
+const {FormText} = renderer;
```

Comment:
This was mis-identified in the past PR as being a problem. This is written correctly. Calling this out pre-emptively to prevent holding things up.